src/ledger/LedgerTxnTrustLineSQL.cpp

```diff
@@ -423,32 +312,22 @@ LedgerTxnRoot::Impl::dropTrustLines()
     std::string coll = mDatabase.getSimpleCollationClause();
 
     mDatabase.getSession() << "DROP TABLE IF EXISTS trustlines;";
-    mDatabase.getSession()
-        << "CREATE TABLE trustlines"
-        << "("
-        << "accountid    VARCHAR(56) " << coll << " NOT NULL,"
-        << "assettype    INT             NOT NULL,"
-        << "issuer       VARCHAR(56) " << coll << " NOT NULL,"
-        << "assetcode    VARCHAR(12) " << coll << " NOT NULL,"
-        << "tlimit       BIGINT          NOT NULL CHECK (tlimit > 0),"
-           "balance      BIGINT          NOT NULL CHECK (balance >= 0),"
-           "buyingliabilities BIGINT CHECK (buyingliabilities >= 0),"
-           "sellingliabilities BIGINT CHECK (sellingliabilities >= 0),"
-           "flags        INT             NOT NULL,"
-           "lastmodified INT             NOT NULL,"
-           "extension    TEXT,"
-           "ledgerext    TEXT            NOT NULL,"
-           "PRIMARY KEY  (accountid, issuer, assetcode)"
-           ");";
+    mDatabase.getSession() << "CREATE TABLE trustlines"
+                           << "("
+                           << "accountid    VARCHAR(56) " << coll
```

Comment:
Nitpick: maybe clean up the spacing in this block?

Reply:
Cleaned up the spacing a bit.